algorithms/CPlusPlus/Strings/anagram.cpp

```diff
@@ -0,0 +1,29 @@
+//Description : To check whether a string is anagram or not.
+//Anagram: An anagram is a word or phrase formed by rearranging the letters in another word or phrase
+#include<bits/stdc++.h>
+using namespace std;
+
+bool isAnagram(string s1, string s2) {
+    int temp[256] = { 0 };
+    for (int i = 0; i < s1.length(); i++) {
+        temp[s1[i]]++;
+        temp[s2[i]]--;
+
+    }
+    for (int i = 0; i < 256; i++) {
+        if (temp[i] > 0) {
+            return false;
+        }
+    }
+    return true;
```

Comment:
use few comment where the code is hard to understand so it's become user friendly program